stock_scrap_tier_validation/models/stock_scrap.py

```diff
@@ -13,13 +13,13 @@ class StockScrap(models.Model):
     _tier_validation_manual_config = False
 
     def _get_requested_notification_subtype(self):
-        return "stock_scrap_tier_validation.sale_order_tier_validation_requested"
+        return "stock_scrap_tier_validation.stock_scrap_tier_validation_requested"
```

Comment:
Could you please cover this line with tests? Missing tests for this line is the main reason, why typo is still here.
The same about other lines.

Reply:
Coverage for current 17.0 for demonstration: https://app.codecov.io/gh/OCA/stock-logistics-workflow/blob/17.0/stock_scrap_tier_validation%2Fmodels%2Fstock_scrap.py

Reply:
@em230418 Done, thanks!